uiowa/uiowa
# [Card] Added linkit to card field link

Resolves https://github.com/uiowa/uiowa/issues/7228./

# How to test
- Test with P2LB
```
ddev blt ds --site=brand.uiowa.edu && ddev drush @brand.local uli /node/761/revisions
```

- Revert to the pre-V3 page
- Edit a card paragraph
   - Add an image 
   - Add a title
   - Remove image from WYSIWYG
- Convert the page to V3
  - Verify that the `/media/` url is converted into the card

## Test overall card functionality
```
ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli /cards
```
Test out card link functionality with Linkit
Test with `<nolink>`, `<front>`, and any others you can think of

## Todo
### Styling
- Do we need extra styling here from https://github.com/uiowa/uiowa/pull/6742
### Circle button
- Circle button logic needs to be reworked
